### PR TITLE
Patientid

### PR DESCRIPTION
**Pull request by juanto121**

### CODE REVIEW

File: app/domain/urls.ts [hunk 1]
```diff
@@ -76,7 +79,8 @@ const localeAwareUrls: Urls = {
     apiDocs: '/api-docs',
     statistics: '/statistics',
     videocall: '/videollamada',
-    riskClassification: 'clasificacion-del-riesgo'
+    riskClassification: '/clasificacion-del-riesgo',
+    volunteerDoctor: '/medico-voluntario'
```
Comment (ibalejandro) [inline app/domain/urls.ts:83]: Shouldn't this be in english? i.e. /risk-classification and /volunteer-doctor

File: app/domain/urls.ts [hunk 2]
```diff
@@ -89,7 +93,8 @@ const localeAwareUrls: Urls = {
...
```
Comment (ibalejandro) [inline app/domain/urls.ts:97]: Shouldn't this be in english? i.e. /risk-classification and /volunteer-doctor

File: app/domain/urls.ts [hunk 3]
```diff
@@ -115,7 +121,8 @@ const localeAwareUrls: Urls = {
...
```
Comment (ibalejandro) [inline app/domain/urls.ts:125]: Shouldn't this be in english? i.e. /risk-classification and /volunteer-doctor

File: app/domain/urls.ts [hunk 4]
```diff
@@ -128,7 +135,8 @@ const localeAwareUrls: Urls = {
...
```
Comment (ibalejandro) [inline app/domain/urls.ts:139]: Shouldn't this be in english? i.e. /risk-classification and /volunteer-doctor

File: app/domain/urls.ts [hunk 5]
```diff
@@ -271,7 +289,8 @@ const localeAwareUrls: Urls = {
...
```
Comment (ibalejandro) [inline app/domain/urls.ts:293]: Shouldn't this be in english? i.e. /risk-classification and /volunteer-doctor

File: app/repository/VideocallAPI.ts [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import config from '../config';
```
Comment (ibalejandro) [inline app/repository/VideocallAPI.ts]: I think we can just delete this entire file because it is no longer needed.

**Review by ibalejandro — CHANGES_REQUESTED** [review]
Some adjustments in app/domain/urls.ts and app/repository/VideocallAPI.ts.

**Comment by juanto121** [comment]
Hacer merge sin mente cuando se hagan los cambios del idioma
Terminar lo de voluntarios en otra rama

**Pull request merged**